pkg/overlord/handlers.go

```diff
@@ -274,22 +273,26 @@ func findBestServerFromBrowser(workerClients map[string]*WorkerClient, client *B
 func getLatencyMapFromBrowser(workerClients map[string]*WorkerClient, client *BrowserClient) map[*WorkerClient]int64 {
 	workersList := []*WorkerClient{}
 
-	latencyMap := map[*WorkerClient]int64{}
-
-	// addressList is the list of worker addresses
-	addressList := []string{}
+	// oh wow! there are no sets in Go
+	addresses := make(map[string]struct{})
 	for _, workerClient := range workerClients {
 		workersList = append(workersList, workerClient)
-		addressList = append(addressList, workerClient.Address)
+		addresses[workerClient.Address] = struct{}{}
+	}
+
+	// make a comma separated list
+	addressList := ""
+	splitter := ""
+	for address := range addresses {
+		addressList += splitter + address
+		splitter = ","
 	}
 
 	// send this address to user and get back latency
-	log.Println("Send sync", addressList, strings.Join(addressList, ","))
-	data := client.SyncSend(cws.WSPacket{
-		ID:   "checkLatency",
-		Data: strings.Join(addressList, ","),
-	})
+	log.Println("Send sync", addressList)
```

Comment:
Yes, Golang philosophy is to simplify coding :p
Can we create an utitlity function: 
```
// unify will deduplication a list.
func unify(string []string)[]string .
```
So the code can just apply this func directly

so the flow can be just
```
addressList = util.Unify(addressList)
data := client.SyncSend(cws.WSPacket{ID: "checkLatency", Data: strings.Join(addressList, ",")})
```

Reply:
I think I'll pass this one.